core/Pos/Core/Genesis/Testnet.hs

```diff
@@ -0,0 +1,217 @@
+-- | Generation of genesis data for testnet.
+
+module Pos.Core.Genesis.Testnet
```

Comment:
Rename to `Pos.Core.Genesis.Generate` or similar? It's not just about testnet, since mainnet data can be generated too.

Reply:
Right, it's planned and it's exactly the name I proposed in our private chat.

Reply:
That's what I thought after looking at the exports. But after reading through the file I wonder is it really useful for generating mainnet data? I see mentions only of fake avvm, but surely in mainnet there's no "fake" data.

Reply:
Actually, we don't generate mainnet data AFAIU, but we use it for dev mode too, so it's reasonable.